github/actions_artifacts_test.go

```diff
@@ -453,25 +453,19 @@ func TestArtifact_Marshal(t *testing.T) {
 		},
 	}
 
-	want := `{
-		"id": 1,
-		"node_id": "nid",
-		"name": "n",
-		"size_in_bytes": 1,
-		"url": "u",
-		"archive_download_url": "a",
-		"expired": false,
-		"created_at": ` + referenceTimeStr + `,
-		"updated_at": ` + referenceTimeStr + `,
-		"expires_at": ` + referenceTimeStr + `,
-		"workflow_run": {
-			"id": 1,
-			"repository_id": 1,
-			"head_repository_id": 1,
-			"head_branch": "b",
-			"head_sha": "s"
-		}
-	}`
+	want := fmt.Sprint(
+		`{"id":1,`,
+		`"node_id":"nid",`,
+		`"name":"n",`,
+		`"size_in_bytes":1,`,
+		`"url":"u",`,
+		`"archive_download_url":"a",`,
+		`"expired":false,`,
+		`"created_at":`+referenceTimeStr+`,`,
+		`"updated_at":`+referenceTimeStr+`,`,
+		`"expires_at":`+referenceTimeStr+`,`,
+		`"workflow_run":{"id":1,"repository_id":1,"head_repository_id":1,"head_branch":"b","head_sha":"s"}}`,
+	)
```

Comment:
@gmlewis  I was thinking of structuring the expected strings this way. I think it will be easier (or at least less difficult) to read, understand and maintain these values.

What do you think about this?

Reply:
Why not just keep all of the line breaks but remove all but the first and last backticks?

That, to me, seems like a lot less noisy clutter.

Reply:
It sounds great!

We need to remove tabs and line breaks. It goes something like this:

```go
s = strings.Replace(s, "\t", "", -1)
s = strings.Replace(s, "\n", "", -1)
```

Do you think we can put it inside the `testJSONMarshal` function, so that all values passed are already "corrected"?

Reply:
Hmm... that's an interesting idea... How about trying it out and seeing if it can handle all the cases we have?

Reply:
I tried it and all the tests kept passing! 🟢 
I will now apply it to all the other test cases.